sdk/src/types/block/output/delegation.rs

```diff
@@ -334,7 +334,7 @@ impl DelegationOutput {
         {
             return Err(StateTransitionError::MutatedImmutableField);
         }
-        // TODO add end_epoch validation rules
+
```

Comment:
Can you remove the `()` from `!next_state.delegation_id()` L327 pls this is bugging me